output/filewriter.go

```diff
@@ -219,106 +383,36 @@ func (fw *fileWorker) makePath() string {
 		"Field0":   fw.replFieldValue,
 	}
 
-	err := fw.pathTemplate.Execute(&doc, replacementVars)
+	err := tmpl.Execute(&buf, replacementVars)
 	if err != nil {
 		panic(err.Error())
```

Comment:
Since the template had already been parsed, en error at `Execute` is a developer error and should be caught asap so i think a panic is right here since a failure can't originate from the user.

However there's an future improvement we could with `PathString` templattng. We could (and we should) at configuration time, call `Execute` once and verify that the corresponding path is valid. For now, if I for example set `pathstring = "/{{.Hello}}"` in the TOML, the generated path is invalid but only sees it when `newWorker` is called